app/views/layouts/application.html.erb

```diff
@@ -1,18 +1,29 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>JakSieMaszSlowfood</title>
     <% if Rails.env.test? %>
       <%= tag.meta id: :fake_position, content: fake_location %>
     <% end %>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.3/css/bootstrap.min.css" integrity="sha384-Zug+QiDoJOrZ5t4lssLdxGhVrurbmBWopoEl+M6BdEfwnCJZtKxi1KgxUyJq13dy" crossorigin="anonymous">
+    <title>JakSieMaszSlowfood</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap-theme.min.css" integrity="sha384-rHyoN1iRsVXV4nD0JutlnGaslCJuC7uwjduW9SVrLvRYooPp2bWYgmgJQIXwl/Sp" crossorigin="anonymous">
```

Comment:
A few things. You are including the css file **twice**. Why? Also, why not use the any of the boostrap gems?

Reply:
css - I was tired and made a mistake. I will change that. 
bootstrap - we, as a group, want links as a reminder and to be conscious which version we are using - sometimes you need to change the version if you want to use another template. But I will bring it up in group meeting for another discussion.